README.md

```diff
@@ -3,15 +3,15 @@
 # Samples for SAP Fiori tools
 
 ### Description
-This project contains sample SAP Fiori elements projects built using SAP Fiori tools. These sample projects will be primarily referenced from blogs and tutorials grouped in folders according to application scenario.
+This repo contains sample SAP Fiori elements projects built using SAP Fiori tools. These sample projects are primarily referenced from blogs and tutorials grouped in folders according to application scenario.
```

Comment:
Suggestion for second sentence: "These sample projects are primarily referenced in blogs and tutorials and are grouped in folders according to application scenario."
```suggestion
This repo contains sample SAP Fiori elements projects built using SAP Fiori tools. These sample projects are primarily referenced from blogs and tutorials grouped in folders according to application scenario.
```
```suggestion
This repo contains sample SAP Fiori elements projects built using SAP Fiori tools. These sample projects are primarily referenced in blogs and tutorials and are grouped in folders according to application scenario.
```